lib/mongoid/history/trackable.rb

```diff
@@ -282,6 +282,8 @@ def track_history_for_action(action)
 
           clear_trackable_memoization
 
+          return unless block_given?
```

Comment:
Do add `&_block` as a parameter in the function definition to make it clear that this function allows for a block.

Reply:
Ok.